Gabriele/operators.js

```diff
@@ -0,0 +1,44 @@
+// Declare two variables "snackServings" and "guests" and assign them number values.
+
+let snackServings = 30;
+let guests = 20;
+
+console.log(snackServings, guests);
+console.log(snackServings + guests);
+
+console.log(snackServings - guests);
+
+console.log(guests - snackServings);
+
+console.log(snackServings * guests);
+
+console.log(snackServings / guests);
+
+let drinks = 10;
+console.log(snackServings * guests / drinks);
+let resultOne = 60;
+console.log(resultOne);
+
+let monkeys = 15;
+let bananas = 9;
+console.log(monkeys / bananas);
+
+let monkeyfights = 20;
+console.log((monkeys + bananas) * monkeyfights);
+let resultTwo = 480;
+console.log(resultTwo);
+
+
+monkeys++;
+console.log(monkeys++);
+
+bananas--;
+console.log(bananas--);
+
+console.log(bananas - monkeys);
+
+let happiness = -10;
+console.log(happiness + monkeyfights);
+
+//console.log(resultOne / resultTwo);
```

Comment:
//11. Print the remainder (%) when resultOne is divided by resultTwo to the console.